src/error.rs

```diff
@@ -586,6 +590,47 @@ impl From<JsonError> for JsonPayloadError {
     }
 }
 
+/// A set of errors that can occur during parsing json payloads
+#[derive(Fail, Debug)]
+pub enum FormPayloadError {
```

Comment:
where do you use this error?

Reply:
oh sorry. I thought it was necessary because it was implemented for JsonError. I am deleting it.

Reply:
Form extractor uses `UrlencodedError` error